main.py

```diff
@@ -43,6 +45,8 @@
         headers=headers,
         offset_minutes=offset_minutes,
         max_retries=max_retries,
+        client_id=csrf_client_id,
+        csrf_enabled = str(os.getenv("PREFECT_CSRF_ENABLED", "False")),
```

Comment:
minor suggestion, but you can make this a boolean before you pass it into the class

```suggestion
        csrf_enabled=str(os.getenv("PREFECT_CSRF_ENABLED", "False")) == "True",
```